src/modules/analysis/AnalysisAddComponent.js

```diff
@@ -142,6 +143,42 @@ const AnalysisAdd = (
     <PageHeader style={{ paddingLeft: 20 }}>
       {I18n.t('musit.analysis.registeringAnalysis')}
     </PageHeader>
+    <Form horizontal style={{ paddingLeft: 10 }}>
+      <FormGroup>
+        <Label label="Type analyse" md={1} />
+        <Col md={2}>
+          <select
```

Comment:
i can of course agree that this is to be done in a different PR. But make a new jira issue for this, where you link the new issue to this one aka 1039. E.g. link with "discovered in" or sumtin